src/main/java/com/redhat/cloud/custompolicies/app/model/Policy.java

```diff
@@ -47,7 +48,12 @@
 
 
   public static Page<Policy> pagePoliciesForCustomer(String customer, Pager pager) {
-    PanacheQuery<Policy> query = find("customerid", customer)
+
+    for (Sort.Column column : pager.getSort().getColumns()) {
+      SortableColumn.fromName(column.getName());
+    }
```

Comment:
@pilhuhn Is this what you had in mind? 
Other alternative that i was thinking is to add an object describing what the `PagingUtils` should allow, e.g.
Allow sorting by a, b and c, allow filtering of b, etc.
And so the RestService would be in charge of describing their needs depending on the service and maybe the user access.

Reply:
I think this is good now.

Reply:
OK, I'll fix the CI error to make travis happy again.